app/assets/javascripts/error.js

```diff
@@ -0,0 +1,22 @@
+pages.error.glassMovement = function(){
+  if ($('body').width() > 880) {
+    $('.error-img').parallax({
+      yparallax: '25px',
+      xparallax: '25px',
+      xorigin: 0,
+      yorigin: 0.9
+    });
+  }
+  $(window).on('resize', function(){
+    if ($('body').width() > 880) {
+      $('.error-img').parallax({
+        yparallax: '25px',
+        xparallax: '25px',
+        xorigin: 0,
+        yorigin: 0.9 });
+    } else {
+      $('.error-img').parallax({ yparallax: false, xparallax: false, });
```

Comment:
You could chain methods ...
```
$('.error-img').
  parallax(...).
  css(...)
```
but ... why are you changing css properties? it would be better to toggle a class with the values ...